src/mpid/ch4/netmod/include/netmod_am_fallback.h

```diff
@@ -11,5 +11,6 @@
 #include "netmod_am_fallback_probe.h"
 #include "netmod_am_fallback_rma.h"
 #include "netmod_am_fallback_coll.h"
+#include "netmod_am_fallback_part.h"
```

Comment:
The commit message should be "ch4/nm: add partitioned comm in netmod am *fallback*

Reply:
done